code/modules/departments/standarddepartments.dm

```diff
@@ -0,0 +1,53 @@
+/datum/department/engineering
+	title = "Engineering"
+	refrence = "engineering"
```

Comment:
`refrence` => `reference`

Reply:
Well at least my misspelling is consistent.